### PR TITLE
Math-magician setup

### PR DESCRIPTION
**Pull request by Kccendana**
### **In this Pull Request, the following was done:**

- Created initial environment and essential tools essential for the development of the Math Magicians application

### CODE REVIEW

File: MIT.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [YOUR NAME]
+
+###### Please delete this line and the next one
+###### APP TYPE can be a webpage/website, a web app, a software and so on
+
```
Comment (Gambit142) [inline MIT.md]: - [ ] Based on the LICENSE template used, you ought to delete these two lines of instruction. Kindly delete them and personalize this LICENSE template by doing the following:

- The text _**'YOUR NAME'**_ should be replaced by your actual name 👍 
- All occurrences of the text _**'APP TYPE**_ should be replaced by the text _**'WEBPAGE'**_ or _**'WEBSITE'**_. 😎 

By doing these, conditions for contributing or using the codes for this project is clearly spelt out to potential contributors. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @Kccendana 👋, 

Good job so far!

## Highlights
- Setup linters correctly ✔️ 
- Application was set up by using **_Create React App_** command ✔️ 
- Application runs in the browser without errors ✔️ 

You still need to work on some issues to go to the next project, but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

You can also consider the following:

N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: MIT.md (hunk 1 above)
Comment (Kccendana) [reply MIT.md]: I updated the license file and added my name and the website name

**Review by taldr27 — APPROVED** [review]
# Status: Approved! 🟢 
Hi @Kccendana, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉

congrats

## Highlights 🎉 
✅ Professional and customized README. 
✅ Linters tests passing.
✅ Correct use of GitHub Flow.
✅ Nice job with your project setup.
✅ Incorporates feedback from previous code reviews.

-----
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

**Pull request merged**